examples/geometry/uniffi.toml

```diff
@@ -0,0 +1 @@
+[bindings.swift]
```

Comment:
There's nothing to do with the swift bindings, but look, the kotlin ones don't need to be there and everything still works.